plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go

```diff
@@ -0,0 +1,259 @@
+package yandex_cloud_monitoring
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+// YandexCloudMonitoring allows publishing of metrics to the Yandex Cloud Monitoring custom metrics
+// service
+type YandexCloudMonitoring struct {
+	Timeout     internal.Duration `toml:"timeout"`
+	EndpointUrl string            `toml:"endpoint_url"`
+	Service     string            `toml:"service"`
+
+	Log telegraf.Logger
+
+	MetadataTokenUrl       string
+	MetadataFolderUrl      string
```

Comment:
some of the linters will complain about this.
```suggestion
	MetadataTokenURL       string
	MetadataFolderURL      string
```

Reply:
Do you mean lexical order of the properties? 
Ok I will change this

Reply:
no, just that URL is typically capitalized

Reply:
Oh. That's right, thank you

Reply:
Done